internal/server/authn/oidc.go

```diff
@@ -202,3 +291,22 @@ func (o *oidcImplementation) GetUserInfo(providerUser *models.ProviderUser) (*In
 
 	return claims, nil
 }
+
+// UpdateUserInfoFromProvider calls the user info endpoint of an external identity provider to see a user's current attributes
+func UpdateUserInfoFromProvider(db *gorm.DB, info *InfoClaims, user *models.Identity, provider *models.Provider) error {
+	// add user to groups they are currently in
+	var groups []string
+
+	for i := range info.Groups {
+		name := info.Groups[i]
+		groups = append(groups, name)
+	}
+
```

Comment:
why not just reference info.Groups? these slices are identical

Reply:
Good spot, I think this is leftover from something in the past. Cleaned it up.